media/js/mozorg/contact-spaces.js

```diff
@@ -408,6 +409,7 @@
             var state = mozMap.getMapState();
             if (state === 'spaces') {
                 initContentId = $('#nav-spaces li.current').data('id');
+                mozMap.showSpace(state.url, initContentId);
```

Comment:
Hmmmm, so I only see requests being made for spaces when it has not already been loaded. Are you seeing it always make a request to load the content or, are you referring to a different request?